packages/strapi/lib/core-api/service.js

```diff
@@ -9,14 +9,42 @@ const {
   },
 } = require('strapi-utils');
 
-const getFetchParams = params => {
+/**
+ * Default limit values from config
+ * @return {{maxLimit: number, defaultLimit: number}}
+ */
+const getLimitConfigDefaults = () => ({
+  defaultLimit: _.toNumber(strapi.config.get('server.rest.defaultLimit', 100)),
+  maxLimit: _.toNumber(strapi.config.get('server.rest.maxLimit', -1)), // (-1) unlimited by default
```

Comment:
We should move these to the `./config/api.js` not the server.
https://strapi.io/documentation/v3.x/concepts/configurations.html#api

Reply:
Config moved to `api.js` and added docs. Maybe it could be rephrased better, what do you think?